engine/api/auth_local.go

```diff
@@ -150,7 +150,7 @@ func initBuiltinConsumersFromStartupConfig(ctx context.Context, tx gorpmapper.Sq
 			Data:               map[string]string{},
 			GroupIDs:           []int64{group.SharedInfraGroup.ID},
 			ScopeDetails:       scopes,
-			IssuedAt:           time.Unix(startupConfig.IAT, 0),
+			ValidityPeriods:    sdk.NewAuthConsumerValidityPeriod(time.Unix(startupConfig.IAT, 0), 0),
```

Comment:
Set a value for services consumers validity period that can be greater than other consumers. Maybe we should update doc for services setup where 'consumer new' command is used.